chainer/functions/math/det.py

```diff
@@ -0,0 +1,106 @@
+import numpy.linalg
+
+from chainer import cuda
+from chainer import function
+from chainer.functions.array.reshape import Reshape
```

Comment:
[OpenStack Style Guideline](http://docs.openstack.org/developer/hacking/) which Chainer use as a coding style, forbids import of objects. So please write `from Chainer.functions.array import reshape` .